src/TypeScriptToKotlinWalker.kt

```diff
@@ -273,8 +273,10 @@ class TypeScriptToKotlinWalker(
                                 unsupportedNode(node)
                         }
 
-        exportedByAssignment[exportName] =
-                Annotation(MODULE, if (moduleName == null) listOf() else listOf(Argument(value = "\"$moduleName\"")))
+        // Kotlin doesn't need to export and import since it uses packages and visibility modifiers.
```

Comment:
But how about tests? Sorry, but we can't disable it in the master. We should somehow map this TS construction to Kotlin.